api/src/main/java/org/openmrs/module/idgen/ConstantPrefixProvider.java

```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.idgen;
```

Comment:
Perhaps for clarity and better code organisation "visually", move all the prefix provider related classes to a subpackage, like `org.openmrs.module.idgen.prefixprovider`.